iceoryx_examples/icedelivery_in_c/README.md

```diff
@@ -17,151 +17,163 @@ so that we explain here only the C API differences and not the underlying mechan
+
+<!--[geoffrey][iceoryx_examples/icedelivery_in_c/ice_c_subscriber.c][create runtime instance]-->
+```c
+const char APP_NAME[] = "iox-c-subscriber";
+iox_runtime_init(APP_NAME);
+```
 
-    ```c
-    const char APP_NAME[] = "iox-c-subscriber";
-    iox_runtime_init(APP_NAME);
-    ```
-  
  2. We create a subscriber port and subscribe to the service
-    {"Radar", "FrontLeft", "Counter" }. Hereby the `historyRequest`
-    tells the subscriber how many previously sent samples it should receive
-    right after the connection is established and the `queueCapacity` how many
-    samples the subscriber can hold. These are samples which the publisher has
-    sent before the subscriber was connected. The `nodeName` is the name of the
-    node, where the subscriber belongs.
+    {"Radar", "FrontLeft", "Object" }. We also set subscriber options. The
+    `historyRequest` tells the subscriber how many previously sent samples it
+    should receive right after the connection has been established and the
+    `queueCapacity` how many samples the subscriber can hold. These are samples
+    which the publisher has sent before the subscriber was connected. The
+    `nodeName` is the name of the node, where the subscriber belongs.
     The `subscriberStorage` is the place where the subscriber is stored in
     memory and `subscriber` is actually a pointer to that location.
 
-    ```c
-    iox_sub_options_t options;
-    iox_sub_options_init(&options);
-    options.historyRequest = 10U;
-    options.queueCapacity = 5U;
-    options.nodeName = "iox-c-subscriber-node";
-
-    iox_sub_storage_t subscriberStorage;
-    iox_sub_t subscriber = iox_sub_init(&subscriberStorage, "Radar", "FrontLeft", "Object", &options);
-    ```
-
- 3. In this loop we receive samples as long the `killswitch` is not
-    set to `true` by an external signal and then print the counter
-    value to the console.
-
-    ```c
-    while (!killswitch)
+<!--[geoffrey][iceoryx_examples/icedelivery_in_c/ice_c_subscriber.c][create subscriber port]-->
+```c
+iox_sub_options_t options;
+iox_sub_options_init(&options);
+options.historyRequest = 10U;
+options.queueCapacity = 5U;
+options.nodeName = "iox-c-subscriber-node";
+iox_sub_storage_t subscriberStorage;
+
+iox_sub_t subscriber = iox_sub_init(&subscriberStorage, "Radar", "FrontLeft", "Object", &options);
+```
+
+ 3. In a loop we receive samples and print the received data on the console as
+    long as the `killswitch` is not set to `true` by an external signal.
+
+<!--[geoffrey][iceoryx_examples/icedelivery_in_c/ice_c_subscriber.c][receive and print data]-->
+```c
+while (!killswitch)
+{
+    if (SubscribeState_SUBSCRIBED == iox_sub_get_subscription_state(subscriber))
     {
-        if (SubscribeState_SUBSCRIBED == iox_sub_get_subscription_state(subscriber))
+        const void* userPayload = NULL;
+        // we will receive here more then one sample since the publisher is sending a
```

Comment:
```suggestion
        // we will receive more then one sample here since the publisher is sending a
```

If suggestion is accepted the code has to be changed ...